ui/js/component/iconLocal/view.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Icon } from "component/common.js";
+
+const IconLocal = props => {
```

Comment:
Did you see my comment on the previous PR raising concerns about this pattern? It seems to scale poorly.

I was suggesting that perhaps icon titles be handled by `Icon` itself with the titles determined by the icon class, or a constant.

Reply:
I saw your comment and then promptly forgot it. We were talking about a lot of small fiddly changes to icons, so it went in one ear and out the other.

It would make sense for me to do that change in this PR. I would think that constants would be better, since it moves the logic into props where it's easy to track. Inspecting CSS classes doesn't feel very React/Redux-y.

Reply:
Ok, please refactor this.